app/controllers/admin/tutorials_controller.rb

```diff
@@ -4,6 +4,15 @@ def edit
   end
 
   def create
+    @tutorial = Tutorial.new(tutorial_params)
+    if @tutorial.valid?
```

Comment:
I've never seen this `.valid?` method. Is there a benefit over calling: `if @tutorial.save`?  .save is the convention used in other places of the app

Reply:
Changed